test-cloud/test-cloud-remote/pom.xml

```diff
@@ -125,6 +130,84 @@
         </dependency>
       </dependencies>
     </profile>
+    <profile>
+      <id>openshift-apb</id>
+      <properties>
+        <skipTests>false</skipTests>
+        <custom.trusted.app.secret>file://${project.build.testOutputDirectory}/secret/rhpam-certs.secret.yaml</custom.trusted.app.secret>
+        <custom.trusted.secret.name>rhpam-certs</custom.trusted.secret.name>
+        <custom.trusted.keystore.alias>jboss</custom.trusted.keystore.alias>
+        <custom.trusted.keystore.pwd>test</custom.trusted.keystore.pwd>
+      </properties>
+      <dependencies>
+        <dependency>
+          <groupId>org.kie.cloud</groupId>
+          <artifactId>framework-openshift</artifactId>
+          <scope>test</scope>
+        </dependency>
+        <dependency>
+          <groupId>org.kie.cloud</groupId>
+          <artifactId>framework-openshift-apb</artifactId>
+          <scope>test</scope>
+        </dependency>
+      </dependencies>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>org.codehaus.mojo</groupId>
+              <artifactId>keytool-maven-plugin</artifactId>
+              <executions>
+                <execution>
+                  <id>importCustomCertificate</id>
+                  <goals>
+                    <goal>importCertificate</goal>
+                  </goals>
+                  <phase>package</phase>
+                  <configuration>
+                    <!--<keystore>${project.build.testOutputDirectory}/certificate/test-keystore.jks</keystore>
+                    <storepass>test</storepass>
+                    <keypass>test</keypass>-->
+                    <storepass>changeit</storepass>
+                    <keystore>${certificate.dir}/test-keystore</keystore>
+                    <keypass>changeit</keypass>
+                    <alias>jboss</alias>
+                    <file>${certificate.dir}/rhpam-test-ca.crt</file>
+                    <trustcacerts>true</trustcacerts>
+                    <noprompt>true</noprompt>
+                  </configuration>
+                </execution>
+                <execution>
+                  <id>importJavaKeystore</id>
+                  <goals>
+                    <goal>importKeystore</goal>
+                  </goals>
+                  <phase>package</phase>
+                  <configuration>
+                    <srckeystore>${java.home}/lib/security/cacerts</srckeystore>
```

Comment:
What certificates are expected to be in this keystore?

Reply:
In this keystore are certificates for java. Newly created certificate in previous execution is not able to communicate with all system.